### PR TITLE
Issue #3375 Websocket example not working

### LINKED ISSUE #3375: Websocket example not working - Throwing Unexpected server response: 500
**Describe the bug**
Since there's no client available for Node.js, I tried to using WebSocket API for creating producers/consumers. When I tried the example given in the documentation, it throws me the "Unexpected server response: 500" error.

**To Reproduce**
1. Run "Pulsar standalone" in docker container(WebSocket service is enabled by default in standalone mode)
2. Check the service is working fine by checking the admin REST APIs
3. Run the Websocket (node.js) examples given [here](https://pulsar.apache.org/docs/en/client-libraries-websocket/#nodejs-producer).
4. You will be able to see the error.

**Expected behavior**
Should be able produce/consume messages using Websocket clients.

**Screenshots**
pulsar-websocket-error

**Desktop (please complete the following information):**
 - OS: Windows 10(but running pulsar in docker container)

**Additional context**
No

### PR DESCRIPTION
**Pull request by ambition119**
### Motivation

Fixes #3375

### Modifications

add Pulsar tenant and namespace created 

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API: (yes / **no**)
  - The schema: (yes / **no** / don't know)
  - The default values of configurations: (yes / **no**)
  - The wire protocol: (yes / **no**)
  - The rest endpoints: (yes / **no**)
  - The admin cli options: (yes / **no**)
  - Anything that affects deployment: (yes / **no** / don't know)

### Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / **not documented**)

### CODE REVIEW

File: site/ja/clients/WebSocket.md [hunk 1]
```diff
@@ -275,20 +275,19 @@ var message = {
 };
 
 ws.on('open', function() {
-  // 1つのメッセージを送信します
-  ws.send(JSON.stringify(message));
+  // Send one message
```
Comment (sijie) [inline site/ja/clients/WebSocket.md]: this is a Japanese documentation page. you don't need to change it :)

Comment (ambition119) [reply site/ja/clients/WebSocket.md]: > this is a Japanese documentation page. you don't need to change it :)

ok

**Comment by sijie** [comment]
run integration tests

**Comment by ambition119** [comment]
run integration tests
```java
ERROR docker[confluentinc/cp-kafka:4.0.0] - Could not start container
```

**Pull request merged**